hack/create-git-release.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -euo pipefail
+
+REPODIR="$(dirname "$0")/.."
+
+if [[ ! -f "${REPODIR}/release" ]]; then
+  echo "Release name file (release) does not exist. Nothing to do"
+fi
+if [[ ! -f "${REPODIR}/release-date" ]]; then
+  echo "Release date file (release-date) does not exist. Nothing to do"
+fi
+
+RELEASE="$(cat "${REPODIR}/release")"
+RELEASE_DATE="$(cat "${REPODIR}/release-date")"
+RELEASE_TAG="v${RELEASE}-${RELEASE_DATE}"
+RELEASE_BRANCH="release-${RELEASE}"
+
+# if the branch corresponding to the release doesn't exist, skip because 
+# it's likely the master branch
+if ! git rev-parse --verify "${RELEASE_BRANCH}" &> /dev/null; then
```

Comment:
We should probably verify that we only create releases from the matching branch.

Reply:
Added a check to ensure that the branch name corresponds to the expected release branch